app/controllers/auth_controller.ts

```diff
@@ -133,4 +136,26 @@ export default class AuthController {
     passwordReset.used = true;
     await passwordReset.save();
   }
+
+  /**
+   * @verifyCaptcha
+   * @operationId verifyCaptcha
+   * @description Checks if token from frontend passes the checks from hCaptcha.
+   * @tag auth
+   * @responseBody 200 - {"status":[ { "message": "Passed check" }]}
+   * @responseBody 401 - {"errors":[ { "message": "Failed check" }]}
+   */
+  async verifyCaptcha({ request, response }: HttpContext) {
+    const { token } = await request.validateUsing(captchaValidator);
+    const remoteIp = request.ip();
+
+    const verdict = await verify(env.get("HCAPTCHA_SECRET"), token, remoteIp);
```

Comment:
i checked out the lib - it's literally just a fetch (but using legacy node libs)
you might as well write write it yourself based on hcaptcha docs